babel/index.babel

```diff
@@ -0,0 +1,27 @@
+// !! IMPORTANT README:
+
+// You may add additional external JS and CSS as needed to complete the project, however the current external resource MUST remain in place for the tests to work. BABEL must also be left in place. 
```

Comment:
I Kindly suggest you remove this file (the babel), because it is not allowed use libraries in this module (module 1)